src/utils/daos.c

```diff
@@ -870,6 +869,8 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 	return 0;
 
 out_free:
+	/* restore number of entries in array for freeing */
+	ap->props->dpp_nr = DAOS_PROP_ENTRIES_MAX_NR;
```

Comment:
I think NLT is failing because this needs a NULL check, since `ap-props` is only allocated in `case DAOS_PROPERTIES_OPTION`.
```suggestion
	if (ap->props != NULL)
		ap->props->dpp_nr = DAOS_PROP_ENTRIES_MAX_NR;
```

Reply:
Good catch, thanks! Fixed.